tests/fib/fib.j2

```diff
@@ -0,0 +1,95 @@
+{# defualt route#}
+{% if testbed_type == 't1' %}
+0.0.0.0/0 {% for ifname, v in minigraph_neighbors.iteritems() %}
+{% if "T2" in v.name %}{{ '[%d]' % minigraph_port_indices[ifname]}}{% if not loop.last %} {% endif %}{% endif %}{% endfor %}
+
+::/0 {% for ifname, v in minigraph_neighbors.iteritems() %}
+{% if "T2" in v.name %}{{ '[%d]' % minigraph_port_indices[ifname]}}{% if not loop.last %} {% endif %}{% endif %}{% endfor %}
+
+{% elif testbed_type == 't0' or testbed_type == 't0-52'or testbed_type == 't0-64' or testbed_type == 't1-lag' or testbed_type == 't0-64-32' %}
```

Comment:
it does not really work for t0 topology now, as the tor_number has been removed from t0 topology file. https://github.com/Azure/sonic-mgmt/pull/1233

I think we need to revisit the fib test later.

Reply:
I used the same fib.j2, which is used by the ansible playbook. 
Should add a check to run this test only for T1 for now  ?